plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotConnectorSmokeTest.java

```diff
@@ -969,6 +989,146 @@ public void testIntegerType()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testJsonExtractIsNull()
+    {
+        assertThat(query("SELECT lucky_number FROM " + JSON_TABLE + " WHERE JSON_EXTRACT(json, '$.selector') IS NULL"))
+                .isFullyPushedDown();
+
+        assertThat(query("SELECT lucky_number FROM " + JSON_TABLE + " WHERE JSON_EXTRACT_SCALAR(json, '$.selector') IS NULL"))
+                .isFullyPushedDown();
+    }
```

Comment:
Please add other assertions
* json-path which `json` doesn't have, e.g. `$.missing`
* json-path which `json` has null, e.g. `{"test_null": null}` and `$.test_null`

Reply:
To address the 1st point, I replaced `$.selector` with `$.missing` in the 1st 2 assertions.

To address the 2nd point, I added `"test_null": null` to `json` and added a 3rd assertion that uses `json_extract_scalar(json, '$.test_null') is null`.

I tried adding a 4th assertion for `json_extract(json, '$.test_null') is null`, but found that `json_extract` isn't behaving as expected under the smoke test. However when using the trino cli manually I'm seeing that predicate pushdown for `json_extract` is working as expected.

Reply:
The issue with `json_extract(json, '$.test_null') is null` is that it doesn't work when pushdowns are disabled. Because of this limitation, I've reduced the support to only `json_extract_scalar`.